addons/html_builder/static/src/core/building_blocks/builder_range.js

```diff
@@ -28,7 +30,7 @@ export class BuilderRange extends Component {
 
     setup() {
         useBuilderComponent();
-        const { state, commit, preview } = useInputBuilderComponent();
+        const { state, commit, preview } = useInputBuilderComponent({ id: this.props.id });
```

Comment:
@FrancoisGe 
Had to add this to make `getItemValue` work with BuilderRange.

Reply:
Yes we merge this change on friday